tests/integration/standard/test_cluster.py

```diff
@@ -282,15 +292,15 @@ def test_connect_on_keyspace(self):
         session = cluster.connect()
         result = session.execute(
             """
-            INSERT INTO test3rf.test (k, v) VALUES (8889, 8889)
+            INSERT INTO test1rf.test (k, v) VALUES (8889, 8889)
             """)
         self.assertFalse(result)
 
-        result = session.execute("SELECT * FROM test3rf.test")
-        self.assertEqual([(8889, 8889)], result)
+        result = session.execute("SELECT * FROM test1rf.test")
+        self.assertEqual([(8889, 8889)], result, "Rows in ResultSet are {0}".format(result.current_rows))
```

Comment:
Yeah, not often, I'm betting on a race condition on the server, but the error message is something like `[(8889, 8889)] != ResultSet`, so hopefully next time we'll know a little bit more of information